Added en text to community channel (#130)
DaoCloud/DaoCloud-docs: docs/en/community/clusterpedia.md

# Clusterpedia, The Encyclopedia of Kubernetes Clusters

> This name Clusterpedia is inspired by Wikipedia. It is an encyclopedia of multi-cluster to synchronize, search for, and simply control multi-cluster resources. 

Reviewer: ```suggestion
This name Clusterpedia is inspired by Wikipedia. It is an encyclopedia of clusters to synchronize, search for, and easily control multi-cluster resources. 
```

> Clusterpedia can synchronize resources with multiple clusters and provide more powerful search features on the basis of compatibility with Kubernetes OpenAPI to help you effectively get any multi-cluster resource that you are looking for in a quick and easy way.  

Reviewer: ```suggestion
Clusterpedia can synchronize resources with multiple clusters. It also provides more powerful search features on the basis of compatibility with Kubernetes OpenAPI. It helps you effectively get any multi-cluster resource that you are looking for in a quick and easy way.  
```

!!! info

>     The capability of Clusterpedia is not only to search for and view but also simply control resources in the future, just like Wikipedia that supports for editing entries.

Reviewer: ```suggestion
    Clusterpedia can search for, view, and control resources in the future, just like Wikipedia that supports entry editing.
```

## Clusterpedia listed in the CNCF landscape

> In the updated CNCF Cloud Native Landscape, Clusterpedia is listed in the Scheduling & Orchestration section of Orchestration & Management block.

Reviewer: ```suggestion
In the latest CNCF Cloud Native Landscape, Clusterpedia is listed in the Scheduling & Orchestration section of Orchestration & Management block.
```
